.changeset/long-geckos-clap.md

```diff
@@ -0,0 +1,8 @@
+---
+'@swisspost/design-system-styles': minor
+---
+
+- Update padding of tertiary buttons from 24px to 4px
+- Update font-size progression to 12/14/16/18
```

Comment:
The two first ones are important, I think. Is "minor" a good fit? Or perhaps "major"?

Reply:
To me, none of the changes here are breaking or add functionality, so I would consider them all patches.

In the future, if you come across a case where you introduced both a minor and a major change, you can add multiple changesets to your PR.